db/postgres/sqlc/schema.sql

```diff
@@ -0,0 +1,32 @@
+-- This file is used by SQLC to autogenerate the Go code needed by the database driver. 
+-- It contains all tables required for storing user data needed by the Gateway.
+-- See: https://docs.sqlc.dev/en/latest/tutorials/getting-started-postgresql.html#schema-and-queries
+--
+CREATE TYPE rate_limit_capacity_period AS ENUM ('daily', 'weekly', 'monthly');
+CREATE TABLE plans (
+    id SERIAL PRIMARY KEY,
```

Comment:
Should/does this have a foreign key with `account_id` or `gateway_endpoints`?

Is there a row per plan type or for every plan that exists? If it's the type, sholdn't that be the primary key?

Reply:
It's the `type`. I've been advised against using strings as primary keys in the database. 

However AFAIK there is nothing wrong with doing do and I have done this many times before (currently the Portal DB `pay_plans` table uses a the pay plan type string as the PK).

I'm fine with making `type` the PK if you are.

Reply:
Will re-review the data model to have a full understanding on it and will leave new comments.